.github/workflows/release.yml

```
@@ -76,14 +77,15 @@
           git config advice.ignoredHook false
           VERSION=${GITHUB_REF#refs/tags/v}
           git commit -m "chore(release): update CHANGELOG for v$VERSION [skip ci]" -a || echo "No changes to commit"
-          BRANCH=$(echo "${GITHUB_REF}" | sed 's@refs/tags/@@')
-          git push origin HEAD:$BRANCH || echo "Push to repository failed"
 
+          git checkout main || git checkout -b main 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)